public/data/people.json

```diff
@@ -0,0 +1,57 @@
+[{
+        "full": "Jason Salerno",
+        "status": "DeafBlind",
+        "title": "Lead Product / Fullstack Development",
+        "img": "/images/team-jason.jpg"
+    },
+
+    {
+        "full": " Giordon Stark",
+        "status": "Deaf / Hard - of -Hearing",
+        "title": "Fullstack Development",
+        "img": "/images/team - giordon.png"
+    },
+    {
+        "full": "Melissa Manak",
+        "status": "Deaf / Hard - of -Hearing",
```

Comment:
Can we change this from `status` to `identify_id` and another to remapped to `identity.json` with identify id?